### PR TITLE
[enh] inline validation

### PR DESCRIPTION
**Pull request by jeremie-lucca**
Valid & Invalid icons don't match. It will need an update after next icons update.

### CODE REVIEW

File: packages/scss/src/components/inputs/textfield/_textfield.material.scss [hunk 1]
```diff
@@ -120,8 +120,30 @@
 	// STATES
 	// ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
 
-	.textfield-input {
+	// Inline validation
+	&.is-valid {
+		&::before {
+			content: "";
```
Comment (sanlucca) [inline packages/scss/src/components/inputs/textfield/_textfield.material.scss]: Why removing the icon in material ?

Comment (jeremie-lucca) [reply packages/scss/src/components/inputs/textfield/_textfield.material.scss]: Yep, we talked about that with @Clotilde. The idea is to use only the bottom line to give succes / error informations.

Comment (sanlucca) [reply packages/scss/src/components/inputs/textfield/_textfield.material.scss]: Then what is the need for the other mods? Aren't borders enough in these cases ?

**Comment by jeremie-lucca** [comment]
image

**Pull request merged**